spec/models/client_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Client, type: :model do
```

Comment:
I prefer to delete files like that if you don't need to test any methods of `Client` class.If you decide to do it later -> you can always create the file then.